deploy_ocp3_cluster.yml

```diff
@@ -13,11 +13,23 @@
   vars_files:
     - "{{ playbook_dir }}/config/defaults.yml"
 
-- name: Run openshift-ansible prerequisites
+- name: Run the openshift-ansible prerequisites for OCP-3.9-3.11
   import_playbook: "{{ lookup('env', 'WORKSPACE') or '../'}}/openshift-ansible/playbooks/prerequisites.yml"
+  vars:
+    ocp: "{{ lookup('env', 'OPENSHIFT_VERSION') }}"
+  when: ocp == "v3.11"
 
-- name: Run the openshift-ansible depoyment
+- name: Run the openshift-ansible deployment for OCP-3.9-3.11
   import_playbook: "{{ lookup('env', 'WORKSPACE') or '../'}}/openshift-ansible/playbooks/deploy_cluster.yml"
+  vars:
+    ocp: "{{ lookup('env', 'OPENSHIFT_VERSION') }}"
+  when: ocp == "v3.11"
```

Comment:
This line will not work for ocp 3.9 and 3.10, it is better to change it to something like
`when: ocp != "v3.7"`, same thing for line 20.

Reply:
done